src/Serilog.Sinks.Map/Serilog.Sinks.Map.csproj

```diff
@@ -1,32 +1,43 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <Description>A Serilog sink wrapper that dispatches events based on a property value.</Description>
-    <VersionPrefix>1.0.3</VersionPrefix>
-    <RootNamespace>Serilog</RootNamespace>
-    <Authors>Serilog Contributors</Authors>
-    <Copyright>Copyright © Serilog Contributors</Copyright>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;net5.0</TargetFrameworks>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
-    <SignAssembly>true</SignAssembly>
-    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
-    <PackageTags>serilog</PackageTags>
-    <PackageIcon>icon.png</PackageIcon>
-    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <PackageProjectUrl>https://github.com/serilog/serilog-sinks-map</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/serilog/serilog-sinks-map</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
-    <GenerateAssemblyVersionAttribute>true</GenerateAssemblyVersionAttribute>
-  </PropertyGroup>
-  
-  <ItemGroup>
-    <PackageReference Include="Serilog" Version="2.8.0" />
-  </ItemGroup>
+    <PropertyGroup>
+        <Description>A Serilog sink wrapper that dispatches events based on a property value.</Description>
+        <VersionPrefix>2.0.0</VersionPrefix>
+        <RootNamespace>Serilog</RootNamespace>
+        <Authors>Serilog Contributors</Authors>
+        <Copyright>Copyright © Serilog Contributors</Copyright>
+        <!-- .NET Framework version targeting is frozen at these two TFMs. -->
+        <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net471;net462</TargetFrameworks>
+        <!-- Policy is to trim TFM-specific builds to `netstandard2.0`, `net6.0`,
+        all active LTS versions, and optionally the latest RTM version, when releasing new
+        major Serilog versions. -->
+        <TargetFrameworks>$(TargetFrameworks);net8.0;net6.0;netstandard2.0</TargetFrameworks>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
+        <SignAssembly>true</SignAssembly>
+        <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+        <PackageTags>serilog</PackageTags>
+        <PackageIcon>icon.png</PackageIcon>
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/serilog/serilog-sinks-map</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/serilog/serilog-sinks-map</RepositoryUrl>
```

Comment:
trivial, but - can SourceLink get RepositoryUrl itself instead of hard coding it?

Actually, do we want a ```<SymbolPackageFormat>snupkg</SymbolPackageFormat>``` for this one?

Reply:
Fixed, thanks. Glad we're down to the last of these! 😅